packages/0x.js/test/token_wrapper_test.ts

```diff
@@ -515,3 +516,14 @@ describe('TokenWrapper', () => {
     });
 });
 // tslint:disable:max-file-line-count
+
+function addEmptyWalletSubprovider(provider: Provider): Provider {
+    const providerEngine = new Web3ProviderEngine();
+    providerEngine.addProvider(new EmptyWalletSubprovider());
+    const currentSubproviders = (provider as any)._providers;
```

Comment:
Hmm this seems a bit hacky, could we export more out of the `utils/web3_wrapper` to avoid this? Like a `export transportProviderEngineProvider`, bit long in the tooth but it might help avoid an `any` and accessing `_providers`.

Do you think this test is executing some logic that would otherwise be skipped if there was a non-empty account provider? If not it feels like a "just make sure nothing blows up" test case. This would also be caught in all of the tests against testnets where there are no accounts?

Reply:
I agree that this is a bit hacky. We do have a precendent of accessing private methods in tests so it's not out-of-line with our conventions. Think it could be a good candidate for a refactor, but fine for this PR. 

If you feel strongly otherwise, please let me know! Still open to changing my mind if there is a really good reason to.